.pre-commit-config.yaml

```diff
@@ -53,3 +53,10 @@ repos:
       - id: clang-format
         args: [-i]
         exclude: module/fstore/|module/test/e2e/.*\.c$|module/test/e2e/.*\.h$
+  - repo: local
```

Comment:
I'd ideally like to remove the ruff hook above and have it just call `make lint` instead. The problem is that it would require that bpf is either present in the system env or that we create a new env for pre-commit that has bpf installed. Better yet, if we could assume that the docker image is built, we could change it to `make -e CONTAINER_CMD="make lint format" docker` which would always work.

Reply:
This is pretty tricky to do, and it's honestly not that useful for now.